libs/application/templates/children-domicile-transfer/src/dataProviders/ChildrenNationalRegistryProvider.ts

```diff
@@ -33,25 +33,22 @@ export class ChildrenNationalRegistryProvider extends BasicDataProvider {
       })
   }
   handleError() {
-    if (process.env.NODE_ENV === 'development') {
-      return Promise.resolve([
-        {
-          id: '1',
-          name: 'Ólafur Helgi Eiríksson',
-          address: 'Vesturgata 22',
-          postalCode: '101',
-          city: 'Reykjavík',
-        },
-        {
-          id: '2',
-          name: 'Rósa Líf Eiríksdóttir',
-          address: 'Vesturgata 22',
-          postalCode: '101',
-          city: 'Reykjavík',
-        },
-      ])
-    }
-    return Promise.resolve({})
+    return Promise.resolve([
```

Comment:
Do we want to add TODO comments on these so they will be removed before shipping?

Reply:
We have a task in Asana to return actual data. I think that will be enough for now.